lnrpc/rpc.proto

```diff
@@ -2527,13 +2526,11 @@ message ChannelGraph {
     repeated ChannelEdge edges = 2;
 }
 
-enum NodeMetricType {
-    BETWEENNESS_CENTRALITY = 0;
-}
+enum NodeMetricType { BETWEENNESS_CENTRALITY = 0; }
```

Comment:
Is there a reason not to have UNKNOWN =0 here?

Reply:
Would it be useful in your opinion to call this RPC with unknown node metric type?

Reply:
I thought it was best practice for protobuf enums to start off with unknown to get around the no-nil problem, but can't see anything in the docs so nvm :)

Reply:
we use an unknown enum in several places so that uninitialized variables aren't accidentally interpreted as "real" enum values. admittedly though this is typically done on responses rather than requests.

@bhandras i'd be okay with adding an unknown type just to future-proof, it's a one line change now but may prevent some headaches down the road. if we end up exposing these enums in a response we may regret not doing so now.

Reply:
In theory, I'm ok with adding an unknown type and understand concerns when we'd store such value or would need reverse compatibility when responding.

I'd be more interested in an actual use case where we'd use it in this context? Is requesting an unknown metric type something that is useful? Looking at current examples in rpc.proto has similar pattern.

Reply:
> Is requesting an unknown metric type something that is useful?

no, definitely not. but it's nothing to do with intentionally requesting an unknown metric, it's just a safeguard against users passing in uninitialized requests accidentally (or us producing uninitialized responses). it's easier to catch when the zero value doesn't map to a real enum

Reply:
Added.
Would be so nice if proto3 didn't deprecate emptiness checks... :)